apps/api/Dockerfile

```
@@ -1,11 +1,59 @@
-FROM ghcr.io/astral-sh/uv:python3.13-bookworm
+# Stage 1: Builder Stage - Install dependencies
+FROM python:3.13-slim-bookworm AS builder
+
+# Install uv
+ENV UV_VERSION=0.1.49
+RUN pip install uv==${UV_VERSION}
```

Hadolint (2.12.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)